src/v/storage/tests/kvstore_test.cc

```diff
@@ -21,8 +23,25 @@ static void set_configuration(ss::sstring p_name, T v) {
     }).get0();
 }
 
-static storage::kvstore_config get_conf(ss::sstring dir) {
-    return storage::kvstore_config(
+/// Call this at end of tests to avoid leaving garbage
+/// directories behind
+static ss::future<> cleanup_store(ss::sstring& dir) {
+    std::filesystem::path dir_path{dir};
+    return ss::recursive_remove_directory(dir_path);
```

Comment:
nice. btw; do we want to add this to the clean up of partition directories. I saw that the latest transaction prevented folders from being deleted because the `rm` file which was later renamed `tx.snapshot` was left over. 

so we might want to add this ss:recursive delete to the segment path (separate patch) but similar issue.

Reply:
I think @rystsov has a ticket for fixing that GC of the `rm` file.